File: android/app/src/main/java/com/h4413/recyclyon/Model/Association.java
package com.h4413.recyclyon.Model;

import java.io.Serializable;

public class Association implements Serializable {

    public String id;
    public String nom;
    public String description;
    public String logoUrl;

    public Association(String id, String nom, String description, String url) {
        this.logoUrl = url;
        this.id = id;
        this.description = description;
        this.nom = nom;
    }
}

File: app/models/association.js
var mongoose     = require('mongoose');
var Schema       = mongoose.Schema;

var AssociationSchema   = new Schema({
    _id: Schema.Types.ObjectId,
    nom: String,
    rib: Number,
    description: String
});

module.exports = mongoose.model('association', AssociationSchema, 'association');
